app/src/services/net/http.service.js

```diff
@@ -79,7 +79,7 @@ export class HttpServiceProvider {
     */
     if (!r.ok) {
       // 401 Unauthorized
-      if (r.status === 401) {
+      if (r.status === 401 || r.status === 403) {
```

Comment:
Nitpick: Comment says 401 unauthorized. Update it to match the new clause, and maybe add why these are caught and what it does?